numpydoc/docscrape_sphinx.py

```diff
@@ -79,21 +79,74 @@ def _str_returns(self, name='Returns'):
                 out += ['']
         return out
 
-    def _str_param_list(self, name):
+    def _str_param_list(self, name, fake_autosummary=False):
         out = []
         if self[name]:
+            if fake_autosummary:
+                prefix = getattr(self, '_name', '')
+                if prefix:
+                    autosum_prefix = '~%s.' % prefix
+                    link_prefix = '%s.' % prefix
+                else:
+                    autosum_prefix = ''
+                    link_prefix = ''
+                autosum = []
+
             out += self._str_field_list(name)
             out += ['']
             for param, param_type, desc in self[name]:
+                param = param.strip()
+
+                display_param = '**%s**' % param
+
+                if fake_autosummary:
+                    param_obj = getattr(self._obj, param, None)
+                    if not (callable(param_obj)
+                            or isinstance(param_obj, property)
+                            or inspect.isgetsetdescriptor(param_obj)):
+                        param_obj = None
+                    obj_doc = pydoc.getdoc(param_obj)
+
+                    if param_obj and (obj_doc or not desc):
```

Comment:
@pv, this is the logic for constructing the description when the attribute corresponds to an object. Firstly, above, we only link to the attribute's autodoc if it has either a docstring or is not described in the numpydoc (and I'd be okay with removing the `or not desc`, except that it is current behaviour).

If it has a docstring, we try to emulate autosummary: split the object's docstring at its first blank line. Then take a prefix up to the first `.` if present, or the whole first paragraph otherwise.

I should probably test this logic more thoroughly before merge.